src/lkl/setup.c

```diff
@@ -31,6 +42,7 @@
 #include "enclave/sgxlkl_t.h"
 #include "enclave/wireguard.h"
 #include "enclave/wireguard_util.h"
+#include "pthread.h"
```

Comment:
What is this for?  We shouldn't need pthreads here...

Reply:
Not sure. I don't remember adding it. I'll remove.